src/de.rs

```diff
@@ -251,6 +293,11 @@ impl<'de, R: Read<'de>> Deserializer<R> {
                 Some(b' ') | Some(b'\n') | Some(b'\t') | Some(b'\r') => {
                     self.eat_char();
                 }
+                // Treat comments as whitespace.
+                Some(b'/') => {
+                    self.eat_char();
+                    return self.parse_comment();
+                }
```

Comment:
This is the entire integration point into existing code. Given the relatively small integration point I hypothesize that we will be able to maintain this as a floating patch in an automated way, indefinitely.